flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroFormatOptions.java

```diff
@@ -71,5 +70,12 @@ public InlineElement getDescription() {
         }
     }
 
+    @Deprecated
+    public static final ConfigOption<Boolean> AVRO_TIMESTAMP_LEGACY_MAPPING =
```

Comment:
Introducing a  **new** config option with `Deprecated` is a little confused to users, I think we can mark it as `Deprecated` when we decide to to deprecate it.